kwave/utils/kwave_array.py

```diff
@@ -83,6 +90,57 @@ def __post_init__(self):
 
         self.measure = float(self.measure)
 
+    def is_close(self, other: "Element", rtol=1e-05, atol=1e-08, equal_nan=False):
+        """Compares 2 Elements to a certain precision for their numerical
+        fields. This differs from the __eq__ method which requires a perfect
+        match between numerical fields. Works with ints, floats and numpy
```

Comment:
Please specify behavior for non-numerical fields as well in docstring. Namely, is_close allows numerical values to vary within the rtol but only when non-numerical fields are the same. When is_close returns False, it could be due to variations in numerical fields beyond rtol, OR because other properties differ.

Reply:
`is_approximately_equal` might be a more accurate name.

Reply:
Will do. The thinking with `is_close` was that it mirrors numpy's `allclose` and `is_close` but happy to change it to `is_approximately_equal`.